dbms/src/Flash/tests/gtest_limit_executor.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <TestUtils/ExecutorTestUtils.h>
+#include <TestUtils/mockExecutor.h>
+
+namespace DB
+{
+namespace tests
+{
+
+class ExecutorLimitTestRunner : public DB::tests::ExecutorTest
+{
+public:
+    using ColDataType = std::optional<typename TypeTraits<String>::FieldType>;
+    using ColumnWithData = std::vector<ColDataType>;
+
+    void initializeContext() override
+    {
+        ExecutorTest::initializeContext();
+
+        context.addMockTable({db_name, table_name},
+                             {{col_name, TiDB::TP::TypeString}},
+                             {toNullableVec<String>(col_name, col0)});
+    }
+
+    std::shared_ptr<tipb::DAGRequest> getRequest(size_t limit_num)
```

Comment:
move the function into ExecutorTestUtils and rename it into a more explicit name？

Reply:
> move the function into ExecutorTestUtils and rename it into a more explicit name？

1. Building tipb::DAGRequest is needed frequently in gtests for executor. However, the way of building DAGRequest is definitely different between different executors(Even parameters in getRequest are different from each other, and we can get this conclusion after comparing the implementation of getRequest in gtest_limit_executor, gtest_projection_executor and gtest_topn_executor). 

2. I will rename it into a more explicit name.